cmd/openshift-install/waitfor.go

```diff
@@ -46,6 +46,9 @@ func newWaitForBootstrapCompleteCmd() *cobra.Command {
 
 			err = waitForBootstrapComplete(ctx, config, rootOpts.dir)
 			if err != nil {
+				logrus.Info("Use the following commands to gather logs from the cluster")
+				logrus.Info("> ssh core@<bootstrap> -A '/usr/local/bin/installer-gather.sh <control-plane1> <control-plane2> <control-plane3> ...'")
+				logrus.Info("> scp core@<bootstrap>:~/log-bundle.tar.gz .")
```

Comment:
Are these <system> lines filled in for the user? 
If not, can they be? 
If not, they can't be (scope creep), we need to document how to pull these out of some installer file. (may need to denote this as a todo somewhere).

Reply:
> If not, they can't be (scope creep), we need to document how to pull these out of some installer file. (may need to denote this as a todo somewhere).

For `create cluster` runs, we already pull it out of installer files (although see [here][1] with quibbles about how we do that).  But `wait-for bootstrap-complete` can also be run in a UPI setting, and then the installer has no way to determine the bootstrap machine's IP address, it's just using a user-provided kubeconfig to talk to the Kubernetes API, and there may be load balancers, etc. in between.

[1]: https://github.com/openshift/installer/pull/1627#discussion_r275936520

Reply:
I think we need to document this clearly but it makes since. tl;dr in IPI situation we try and auto pillot the collection. In UPI, we show you how to fly, plug in your vectors.

Reply:
Docs have a card to document this.